kite-data/kite-data-hive/src/main/java/org/kitesdk/data/spi/hive/HiveExternalMetadataProvider.java

```diff
@@ -82,6 +83,25 @@ public DatasetDescriptor create(String namespace, String name, DatasetDescriptor
           .build();
     }
 
+    if (descriptor.getSchemaUrl() == null) {
+
+      // No schema URL is provided, so create one based on the dataset.
+      Path managerPath = new Path(new Path(newDescriptor.getLocation()),
+          SCHEMA_DIRECTORY);
```

Comment:
Ah, I didn't understand the desired behavior of the provided schema URL, and had assumed there was some case where maintaining an external reference was desired. But I agree that copying it the manager makes the system more robust and will make that change.